setup.py

```diff
@@ -39,10 +39,10 @@
         'jinja2',
         'joblib',
         'lime',
+        'mpich<3.4',  # issues with 3.4.1 and 3.4.2 that install external_0 build
         'matplotlib>3.3.1,<3.4',  # bug in 3.3.1, issue with nc-time-axis for >3.3.4
         'natsort',
         'nc-time-axis<1.3.1',  # needed by iris.plot, issues with matplotlib 3.4 and 1.3.1
-        'netCDF4',
```

Comment:
Why do you remove netCDF4? I mean, it kind of makes sense, but I want to understand if it is necessary to make things work and why for future knowledge.

Reply:
crap, my bad!

Reply:
No worries. As a dependency of a dependency, it should probably not be here, but let's make this kind of change later on.

Reply:
agreed! let's get this in so Manu can run his shtuff :grin: